src/common/components/resetPasswordModal/resetPasswordModal.component.js

```diff
@@ -38,9 +40,10 @@ class ResetPasswordModalController {
   }
 
   $onDestroy() {
-    angular.forEach( ['e', 'k'], ( key ) => {
-      this.$location.search( key, null );
-    } );
+    this.removeQueryParams();
+    if(!this.passwordChanged && !this.exitWithoutRefresh){
+      this.$window.location.reload();
```

Comment:
This will probably need a $timeout added, firefox has a bug whit this.
See https://github.com/CruGlobal/give-web/blob/89489245caf086945ebe2409032db5df4e4ac408/src/common/components/nav/nav.component.js#L149